src/output.rs

```diff
@@ -145,18 +145,31 @@ pub fn print(duplicates: DashMap<String, Vec<File>>, opts: &Params) {
 
     let mut output_table = Table::new();
     output_table.set_titles(row!["hash", "duplicates"]);
-    duplicates.into_iter().for_each(|(hash, group)| {
-        let mut inner_table = Table::new();
-        inner_table.set_format(*format::consts::FORMAT_NO_BORDER_LINE_SEPARATOR);
-        group.iter().for_each(|file| {
-            inner_table.add_row(row![
-                format_path(&file.path, opts).unwrap_or_default().blue(),
-                file_size(&file.path).unwrap_or_default().red(),
-                modified_time(&file.path).unwrap_or_default().yellow()
-            ]);
+    duplicates
+        .into_iter()
+        .map(|f| {
+            // we extract the file size of the first file in the group
+            let size =
+                f.1.first()
+                    .and_then(|ff| fs::metadata(&ff.path).ok())
```

Comment:
@beeb I made some changes to avoid recalculating the size of files as we do it for the purpose of detecting duplicates right at the beginning. This is causing a conflict. could you resolve the conflict? 
and now you can use ff.size.unwrap_or_default() to get the size of a file as u64. as its passed into the DashMap<String, Vec<File>> arg.

Reply:
OK will do!

Reply:
Thank you!

Reply:
Made my life easier actually, the conflicts have been resolved